irohad/torii/impl/status_bus_impl.cpp

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "torii/impl/status_bus_impl.hpp"
+
+namespace iroha {
+  namespace torii {
+    StatusBusImpl::StatusBusImpl(rxcpp::observe_on_one_worker worker)
+        : worker_(worker), subject_(worker_) {}
```

Comment:
What is the point to hold `worker_` field?

Reply:
If it passed directly to the `subject_` that will create a new thread for each subscriber